app/views/layouts/basic.html.haml

```diff
@@ -1,33 +1,35 @@
-- @disable_logo_link  = true
-
 - content_for?(:page_title) ? yield(:page_title) : contextual_title
 
 - content_for :head do
-  %meta{:name =>"format-detection", :content => "telephone=no"}
-  = stylesheet_link_tag "application", media: "all"
+  %meta{ name: 'format-detection', content: 'telephone=no' }
+  = stylesheet_link_tag 'application', media: 'all'
 
 - content_for :body_classes do
   = "controller-" + controller.controller_name
 
+- content_for :header_class do
+  = "with-proposition"
+
 - content_for :proposition_header do
   .header-proposition
     .content
       %nav#proposition-menu
-        %a#proposition-name{:href =>"/"}
+        %a#proposition-name{:href => "/"}
```

Comment:
Code consistency: You should most likely replace the hash rocker with a colon as you did on line 4

Reply:
done